web/src/components/vis/use/usePlotData.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  computed, Ref, ref, onActivated, onDeactivated, watchEffect,
+} from '@vue/composition-api';
+import store from '../../../store';
+import { LOAD_PLOT } from '../../../store/actions.type';
+import { SET_PLOT } from '../../../store/mutations.type';
+
+export default function useDataPlot(id: Ref<string>, plotName: string) {
```

Comment:
I believe this should be `usePlotData`, right?

```suggestion
export default function usePlotData(id: Ref<string>, plotName: string) {
```